packages/transforms/README.md

```diff
@@ -1 +1,55 @@
+# @endo/transforms
 
+> Source transforms for [SES](https://github.com/endojs/endo/tree/master/packages/ses)-enabled applications
+
+This package currently provides a transform to modify comments which would otherwise be rejected outright by SES.
+
+## Example
+
+```js
+// ESM example
+import { transformSource } from '@endo/transforms';
```

Comment:
I’m imagining that the scope of the `transforms` package allows for additional transforms unrelated to SES censorship evasion. Do you think it might make sense to narrow the scope of the package by making its name more specific, or have a more specific sub-path for SES censorship evasion? Can we avoid a situation where a dependent module has to entrain all of the package’s transforms when it only needs one?

Reply:
Renaming this to e.g., `@endo/evasive-transform`  and limiting scope seems reasonable--though it potentially means adding more pkgs in the future.  At first glance, Endo doesn't seem like a project that fears this sort of tradeoff. 😄

Reply:
For the record, I laughed, and my laugh was audible in an adjacent room, but not audible to our neighbors. Thank you.